core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -451,7 +451,8 @@ object KafkaConfig {
   "it will publish the same port that the broker binds to."
   val AdvertisedListenersDoc = "Listeners to publish to ZooKeeper for clients to use, if different than the listeners above." +
```

Comment:
minor: can we remove the word  "above" from the description.  the description can be like this  "if different than the  "listeners" config property"

LGTM
cc @ijuma

Reply:
Done